### ISSUE TITLE
MeshCentral Assistant non responsive when open system files on Windows 11

### ISSUE THREAD

**Issue by demkr** [Ylianst/MeshCentral#6525]
Bug description:
When connecting to a Win11 host via MeshCentral Assistant, the entire screen becomes unresponsive to mouse and keyboard clicks when I open a system application (via the run command or directly from the system32 directory) and the application window is open. When the user on the remote host closes the window of this system application, everything becomes responsive again. When I did the same thing on Windows 10, everything was OK.

1. Launch the MeshCentral Assistant application on the remote host running Windows 11. When choosing from the invitation codes, only "Application, connection at the request of the user" is available, so that was the version I have used.
2. On Assistant window click "Request help", enter the text, and press OK
3. Connect from the MeshCentral server via the desktop and run some system commands. I tried regedit.exe, mmc.exe, devmgmt.msc
4. When a window (for example, from regedit) is open, the mouse and keyboard on the MeshCentral server are unresponsive and therefore remote administration is not possible.
5. When the remote user closes ,for example, the regedit window, the mouse and keyboard become responsive and remote administration is possible again.

Expected Behavior
Responsiveness of mouse and keyboard should be the same regardless of which application window is open.

MeshCentral Server Software
- OS: Ubuntu 22.04.5 LTS
- MeshCentral Version: 1.0.33
- Node.js Version: 20.17.0

Remote Device - Affected by the Bug
- Device: Laptop
- OS: Windows 11 23H2

Remote Device - Not Affected by the Bug
- Device: Laptop
- OS: Windows 10 22H2

**Comment by si458** [comment]
meshcentral assistant doesnt have admin rights, it runs with user permissions
any applications that run as admin you wont be able to control or access (taskmgr, regedit, etc)

the better way would be to use the meshagent and use the `connect` button,
this then asks for admin rights and runs correctly with admin rights
but in a domain environment it would ask for the admins user/pass which isnt ideal

**Comment by demkr** [comment]
@si458 Yes, I know that the meshcentral assistant does not have administrative rights, or if I understand correctly, it has the same rights as the logged in user. That actually doesn't bother me. What bothers me is that if the session through the assistant did not freeze, I could enter administrative credentials if necessary (let say for UAC). This is how the assistant works in Win 10. However, in Win 11, the session does not respond, even if the user is in the administrators group or the assistant is running as administrator and as long as the system file window is open. When the window is closed, by the remote user, the session becomes responsive again.

**Comment by si458** [comment]
@demkr you cant access the windows UAC prompt or enter details
when you use the 'assistant' only the 'meshagent' can get access to it ?

**Comment by demkr** [comment]
@si458 yes you are right. I also got the same freeze in Win 10 when trying to give credentials in UAC. So I assume there will be no assistant changes regarding this issue and some alternative is to use interactive only agent.

**Comment by demkr** [comment]
@si458 I looked at link you provided and this is very good suggestion. Also I can confirm that your workaround provided there doesnt work on WIN 11 (i didnt test on WIN 10 yet)

**Comment by si458** [comment]
sadly the isnt anything we can do from what im aware,
im not sure how the 'assistant' is coded but from my understanding,
it was made for non-admin access which is great! but also doesnt help when it comes to accessing the 'admin' prompts 
or 'admin' apps or even the UAC,
you should really use the 'meshagent' and use the 'connect' option for like 1 time connect access, 
it just requires the end user to type admin credentials or click 'run as admin' etc

**Comment by si458** [comment]
@demkr yeh i just checked the post, it doesnt work, i have no idea why i said `try this` a year ago?
